index.html

```diff
@@ -79,6 +79,16 @@ <h3 class="img-title">Conversation</h3>
                 <h3 class="img-title">Store</h3>
                 <p class="img-des">The most variable selection of books</p>
             </div>
+            <div class="item-list">
+                <img src="./images/list4.png" alt="buy" class="images">
+                <h3 class="img-title">Reviews</h3>
+                <p class="img-des">A little checks of the best sellers books</p>
+            </div>
+            <div class="item-list">
+                <img src="./images/list5.png" alt="buy" class="images">
+                <h3 class="img-title">Exhibition</h3>
+                <p class="img-des">Exhibition of a new novel to Haruki Murakami</p>
+            </div>
```

Comment:
- [ ] Kindly fix the `See the whole program` button in the desktop view. It is supposed to be an underlined text in the desktop view. It would be great if you could correct that 👍 
- Your design:
![Screenshot 2022-07-02 054418](https://user-images.githubusercontent.com/85113567/176986849-d15670dc-5274-448d-82c8-27116e6be6a1.png)
- Required design:
![Screenshot 2022-07-02 054438](https://user-images.githubusercontent.com/85113567/176986855-ad9984ac-83d5-439e-8f49-02b9d6f0280a.png)